### ISSUE TITLE
Doesn't work on IE11 

### ISSUE THREAD

**Issue by rajbak** [derrickpelletier/react-loading-overlay#30]
This component doesn't work on IE11. It crashes when `active` prop switches from `true` to `false`.
Didn't have time to debug it, but it looks like this is because the following code:
`css(this.getStyles('wrapper', active && overflowCSS))` for the `active={false}` returns `[{Object}, false]` array and later `Object.keys()` method is called on each item from this array (from `_objectSpread(target)` method). Calling `Object.keys(false)` on IE11 throws en exception, however on Chrome it evaluates to `[]`.

To reproduce run the below code on IE11:

```
<LoadingOverlay
  active={false}
  >
  <p>Some content</p>
</LoadingOverlay>
```

To workaround this you can use below code:
```
<LoadingOverlay active={loading ? true : undefined}>
  <p>Some content</p>
</LoadingOverlay>
```
It works because `_objectSpread(target)` method checks if an item is null/undefined and if so, it replaces it with an empty object `{}`.

**Comment by derrickpelletier** [comment]
Thanks for the details!
If you want to submit a PR it’d be appreciated, otherwise will try to get to it next week 😀

**Comment by derrickpelletier** [comment]
Published as `1.0.1`, thanks
